rdoc/synchtml.sh

```diff
@@ -21,9 +21,13 @@ else
 fi
 
 if [ $docdir != "master" -a $docdir != "html602" ]; then
-   tarname=${docdir/v/html}.tar.gz
-   tar zcf $tarname $docdir
-   scp $tarname root.cern.ch:/var/www/root/download/
+   tarnamegz=${docdir/v/html}.tar.gz
+   tar zcf $tarnamegz $docdir
+   scp $tarnamegz root.cern.ch:/var/www/root/download/
+   tarnamexz=${docdir/v/html}.tar
+   tar cf $tarnamexz $docdir
+   xz -T5 $tarnamexz
```

Comment:
Just do
```suggestion
   tar cJf $tarnamexz $docdir
```
which combines these two lines into one.

Reply:
Instead of doing all in one command I did it in 2 passes (tar first and then xv). in order to specify -T5 which runs xz within 5 threads and speed  a lot the command execution ... (it is super slow otherwise)